src/main/scala/org/broadinstitute/dsde/workbench/avram/dataaccess/HttpSamDao.scala

```diff
@@ -24,23 +23,23 @@ class HttpSamDao(samUrl: String) extends SamDao with RestClient {
     } yield userInfo
   }
 
-  private def responseToErrorResponse(response: Response[String]): String => ErrorResponse = error => ErrorResponse(response.code, error)
-  private def circeErrorToErrorResponse(e: io.circe.Error): ErrorResponse = ErrorResponse(500, e.getMessage)
+  private def responseToErrorResponse(response: Response[String]): String => AvramException = error => AvramException(response.code, error)
+  private def circeErrorToErrorResponse(e: io.circe.Error): AvramException = AvramException(500, e.getMessage)
```

Comment:
i don't like losing track of the stack trace. can we also include the error itself? or at a minimum log it.